sources/packages/backend/apps/api/src/auth/roles.enum.ts

```diff
@@ -9,6 +9,7 @@ export enum Role {
   AESTReports = "aest-reports",
   AESTCreateInstitution = "aest-create-institution",
   AESTEditCASSupplierInfo = "aest-edit-cas-supplier-info",
+  AESTCASInvoicing = "aest-cas-invoicing",
```

Comment:
From what I can see from API perspective is the ministry user requires this role even to view any information on CAS Invoices.

Reply:
My understanding is that even the link in the left-side menu should be disabled, so I assume the entire CAS-related data should not be visible.

Reply:
Reviewing the ACs, it does include the "view" also 😉